api/src/pcapi/routes/backoffice/offers/forms.py

```diff
@@ -251,6 +258,20 @@ def __init__(self, *args: list, **kwargs: dict):
         field_list_compatibility=True,
     )
 
+    music_type = fields.PCSelectMultipleField(
+        "Type de musique",
+        choices=music_types.MUSIC_LABEL,
```

Comment:
Autant ne pas créer de MUSIC_LABEL, et faire
`choices=[(str(s), MUSIC_TYPES_LABEL_BY_CODE[s]) for s in MUSIC_TYPES_LABEL_BY_CODE]`

cf ce que j'ai fait pour `show_sub_type` dans `/backoffice/offer_validation_rules/forms.py`